heapify.mjs

```diff
@@ -153,10 +153,25 @@ export default class Heapify {
     }
 
     toString() {
+        if (this.length - ROOT_INDEX < 0 ) {
+            return '(empty queue)'
```

Comment:
I know this is my fault for not having introduced style check yet, but I've been using double quotes for strings. If you feel like it, please go ahead and replace all single quote string occurrences... otherwise I can do it myself later when I take the time to add a linter.

Reply:
Addressesd ;)